client/src/telemetry/glean-context.tsx

```diff
@@ -213,6 +217,7 @@ export function useGleanPage(pageNotFound: boolean, doc?: Doc) {
       // on port 3000 this will always return "200":
       httpStatus: pageNotFound ? "404" : "200",
       userAgent: navigator?.userAgent,
+      userLanguages: Array.from(navigator.languages),
```

Comment:
Maybe we should be more defensive here, in case some browser returns `undefined` here.